domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3129,43 +3129,23 @@ public void startEmbeddedPostgres() throws IOException {
     // starting Postgres
     setIsEmbedded(true);
     if (postgresProcess == null || !postgresProcess.isProcessRunning()) {
-      // turns off the default functionality of unzipping on every run.
-      IRuntimeConfig runtimeConfig = new RuntimeConfigBuilder()
-        .defaults(Command.Postgres)
-        .artifactStore(
-          new NonCachedPostgresArtifactStoreBuilder().defaults(Command.Postgres).
-          download(new PostgresDownloadConfigBuilder().defaultsForCommand(Command.Postgres)
-          // .progressListener(new LoggingProgressListener(logger, Level.ALL))
-            .build())).build();
-      PostgresStarter<PostgresExecutable, PostgresProcess> runtime = PostgresStarter.getInstance(runtimeConfig);
-
       int port = postgreSQLClientConfig.getInteger(PORT);
       String username = postgreSQLClientConfig.getString(_USERNAME);
       String password = postgreSQLClientConfig.getString(_PASSWORD);
       String database = postgreSQLClientConfig.getString(DATABASE);
 
       String locale = "en_US.UTF-8";
       String OS = System.getProperty("os.name").toLowerCase();
-      if(OS.indexOf("win") >= 0){
+      if (OS.indexOf("win") >= 0) {
         locale = "american_usa";
       }
-
-      final PostgresConfig config = new PostgresConfig(Version.Main.V10,
-        new AbstractPostgresConfig.Net(DEFAULT_IP, port),
-        new AbstractPostgresConfig.Storage(database),
-        new AbstractPostgresConfig.Timeout(20000),
-        new AbstractPostgresConfig.Credentials(username, password));
-
-      config.getAdditionalInitDbParams().addAll(Arrays.asList(
-        "-E", "UTF-8",
-        "--locale", locale
-      ));
-
-      postgresProcess = runtime.prepare(config).start();
-
-      log.info("embedded postgress started....");
+      EmbeddedPostgres e = new EmbeddedPostgres(Version.Main.V10);
```

Comment:
Avoid single letter variable name e, renaming will make the code more easy to read and understand.